### PR TITLE
Change JWT to Bearer

### PR DESCRIPTION
**Pull request by colescott**

### CODE REVIEW

**Comment by colescott** [comment]
@eliaslfox Does this look good to merge to master?

**Comment by eliaslfox** [comment]
Yo

**Comment by eliaslfox** [comment]
What is express session for>

File: birds/api/v1/index.js [hunk 1]
```diff
@@ -3,74 +3,18 @@ const mongoose = require('mongoose');
...
 
-// Paths not requiring auth. Path is string or regex
-const unprotectedPaths = [
-    {path: '/auth/login', method: 'POST'},
-    {path: '/users', method: 'POST'},
-    {path: '/users', method: 'GET'},
-    {path: /\/users\/\w+/igm, method: 'GET'}
-]
+router.use(passport.initialize());
+router.use(passport.session());
```
Comment (eliaslfox) [inline birds/api/v1/index.js]: Fuck off

File: birds/api/v1/index.js [hunk 2]
```diff
@@ -138,7 +78,7 @@ router.get("/users/:id", (req, res) => {
     });
 });
 
-router.put("/users/:id", (req, res) => {
+router.put("/users/:id", authenticate, (req, res) => {
```
Comment (eliaslfox) [inline birds/api/v1/index.js]: FIX YOUR SHIT

File: birds/api/v1/index.js [hunk 3]
```diff
@@ -198,7 +138,7 @@ router.put("/users/:id", (req, res) => {
         });
 });
 
-router.put("/users/:id/:action", (req, res) => {
+router.put("/users/:id/:action", authenticate, (req, res) => {
```
Comment (eliaslfox) [inline birds/api/v1/index.js]: FIX YOUR SHIT

File: birds/api/v1/index.js [hunk 4]
```diff
@@ -225,30 +165,29 @@ router.put("/users/:id/:action", (req, res) => {
 });
 
 router.post('/auth/login', function(req, res, next) {
-    passport.authenticate('local', function(err, user, info) {
+    passport.authenticate('local', {
```
Comment (eliaslfox) [inline birds/api/v1/index.js]: IDFN

**Pull request merged**